conans/client/build/cmake_flags.py

```diff
@@ -279,68 +301,85 @@ def get_definitions(self):
         runtime = self._ss("compiler.runtime")
         build_type = self._ss("build_type")
 
-        ret = OrderedDict()
-        ret.update(runtime_definition(runtime))
+        defines = OrderedDict([
```

Comment:
I don't get this change. Why initializing all these variables to None?
What is the value to the user or to Conan?